build.gradle

```diff
@@ -38,6 +43,10 @@ repositories {
 		name "Modmuss50 Repository"
 		url 'https://maven.modmuss50.me'
 	}
+	maven {
+		name "Jitpack"
+		url 'https://jitpack.io/'
```

Comment:
Can we drop this? This makes buildscript slow as hell

Reply:
Just out of curiosity does the newest commit make any noticeable difference?